index.html

```diff
@@ -114,7 +114,35 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
-	 
+        
```

Comment:
Looks like you did not include the comments at the start and end of the template.